app/models/petition_creator.rb

```diff
@@ -143,6 +144,10 @@ def location_code
     petition_creator_params[:location_code] || "GB-SCT"
   end
 
+  def privacy_notice
+    petition_creator_params[:privacy_notice] || "0"
```

Comment:
couldn't this be false or true?

Reply:
This is a virtual attribute from the posted param so it matches what `validates_acceptance_of` needs:

https://github.com/rails/rails/blob/130c128eae233bf71231c73b9c3c3b3f3ede918b/activemodel/lib/active_model/validations/acceptance.rb#L7

To use an actual instance of `TrueClass` you'd need to interpret the string from the user via the standard boolean types that Active Record accepts and it's just easier to use `"1"` and `"0"`